playground/mocks/data/idp/idx/error-429-api-limit-exceeded.json

```diff
@@ -0,0 +1,7 @@
+{
+  "errorCode": "E0000047",
```

Comment:
curious why api limit has different structure with `too many request`

Reply:
> curious why api limit has different structure with `too many request`

My 0.02:  API rate-limiting such as this is generic Okta monolith HTTP / REST API functionality, not specific to IDX, so I look at this from the other end of the spectrum -- IMO having any sort of JSON response body is sort of an optional extra, not fundamental, so I wouldn't necessarily expect it to be an IdxResponse.  To me the fundamental pieces are the HTTP 429 response code and the HTTP response headers (not body) that convey rate-limiting information, but at present we can't get at any of those (OKTA-386750);  absent that, this PR has to fall back to trying to glean information from the Okta (but not IDX) JSON response body.